### ISSUE TITLE
The descriptoin of latency (ms) is misleading

### ISSUE THREAD

**Issue by marcingminski** [marcingminski/sqlwatch#47]
reported by @larspl on slack (Thank you)
> the disk latency (ms) thats comes from io_stall_read/write AVG Counters needs to be renamed. is should show the word "stall" in dashboard
> got yesterday a call from a customer : he means that 26523426ms is to high for one IO ......
> i confirm

My interpretion: It is understood that the storage IO latency would be measured per storage transaction and averaged over the measurment period, or, preferably max value taken from the measurement period. However, the dashboard shows cumulative stall across all files and all IO transactions and therefore:
- either description must be changed to reflect this (as suggested by @larspl)
- or, the actual IO latency calculated

I will investigate further.

**Comment by marcingminski** [comment]
I have looked into this as there were some more questions on our slack channel. 

The calculation in PowerBI is a delta between two snapshots:
```sql
[io_stall_read_ms_delta] = ((fs2.io_stall_read_ms - fs1.io_stall_read_ms) + (fs2.io_stall_write_ms - fs1.io_stall_write_ms))
```

However, as @larspl pointed out, this is still delta stall, not latency. As per the [MS explanation](https://docs.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-io-virtual-file-stats-transact-sql?view=sql-server-2017), the stall is: 

> Total time, in milliseconds, that the users waited for reads issued on the file.

so, in order to calculate latency, we need to take the number of operations (writes/reads) into account i.e:

```sql
((fs2.io_stall_read_ms - fs1.io_stall_read_ms) + (fs2.io_stall_write_ms - fs1.io_stall_write_ms)) / ((fs2.num_of_reads - fs1.num_of_reads) + (fs2.num_of_writes - fs1.num_of_writes))
```
So the data collection and calculation **of the stall** is correct, it's just that showing stall on its own is not particulary useful.

I have released a fixed version of the PowerBI report with the actual latency calculation and few more columns to aid debugging in c559fa598573bd1e0f3c6ca7f98940aee9e017d2

**Thank you to all involved for bringing this to my attention and helping fix it**
